docs/concepts/what-is-a-workflow-execution.md

```diff
@@ -2,57 +2,122 @@
 id: what-is-a-workflow-execution
 title: What is a Workflow Execution?
 sidebar_label: Workflow Execution
-description: A Workflow Execution is a Reentrant Process; that is, a resumable, recoverable, and reactive process.
+description: A Temporal Workflow Execution is a durable, scalable, reliable, and reactive function execution. It is a Temporal Application’s main unit of execution.
 tags:
   - explanation
 ---
 
-A Workflow Execution is a Reentrant Process; that is, a resumable, recoverable, and reactive process:
+A Temporal Workflow Execution is a durable, reliable, and scalable function execution.
+It is a Temporal Application’s main unit of execution.
 
-- Resumable: Ability of a process to continue execution after execution was suspended on an await-able.
-- Recoverable: Ability of a process to continue execution after execution was suspended on a failure.
-- Reactive: Ability of a process to react to external events.
+- [How to spawn a Workflow Execution in Go](/docs/go/how-to-spawn-a-workflow-execution-in-go)
+- [How to spawn a Workflow Execution in Java](/docs/java/how-to-spawn-a-workflow-execution-in-java)
+
+Each Temporal Workflow Execution has exclusive access to its local state, executes concurrently to all other Workflow Executions, and communicates with other Workflow Executions and the environment via message passing.
+
+A Temporal Application can consist of millions to billions of Workflow Executions.
```

Comment:
A single workflow execution has limited size and throughput. An application scales out by executing more workflow executions in parallel.